daemon/graphdriver/imagerestorer.go

```diff
@@ -0,0 +1,25 @@
+package graphdriver
+
+import (
+	"github.com/docker/docker/image"
+	"github.com/docker/docker/pkg/archive"
+)
+
```

Comment:
Let's add an appropriate warning message about using these interfaces. These are looking pretty good, but let's indicate that they are for the windows driver implementation.